torch_xla/csrc/tensor_methods.cpp

```diff
@@ -155,31 +155,39 @@ ir::Value GetIrValueOrDefault(const XLATensor& input, at::Scalar default_value,
 }  // namespace
 
 XLATensor XLATensor::__and__(const XLATensor& input, at::Scalar other) {
-  XLA_CHECK(xla::ShapeUtil::ElementIsIntegral(input.shape()))
-      << "Bitwise and is only supported for integer type tensors";
+  XLA_CHECK(XlaHelpers::ElementIsIntegralOrPred(input.shape()))
```

Comment:
This screams for a CheckIntegralOrPred(shape) API in an anon namespace in this file.
Drop the ElementIsIntegralOrPred() API.

Reply:
Yeah, I waffled a bit between helpers and anon namespace. I'll move it.

Reply:
Done.